src/main/java/bio/terra/controller/GlobalExceptionHandler.java

```diff
@@ -74,7 +85,10 @@ private ErrorModel buildErrorModel(Throwable ex) {
         for (Throwable cause = ex; cause != null; cause = cause.getCause()) {
             logger.error("   cause: " + cause.toString());
         }
-        return new ErrorModel().message(ex.getMessage());
+        StringWriter sw = new StringWriter();
+        PrintWriter pw = new PrintWriter(sw);
+        ex.printStackTrace(pw);
+        return new ErrorModel().message(ex.getMessage()).stackTrace(sw.toString());
```

Comment:
I don't think we should use the return through the REST API for our debugging. The error returned should be suitable for display in the UI.

If we aren't catching enough debug info, then we should fix that.
In the integration environment, it should be in the stackdriver logs.

Reply:
yeah, we were not logging enough info before. i'll remove.